--- lib/checksrc.pl_orig	2024-04-28 19:32:54.461689929 +0200
+++ lib/checksrc.pl	2024-04-28 19:32:57.871689836 +0200
@@ -428,10 +428,10 @@
 
         # crude attempt to detect // comments without too many false
         # positives
-        if($l =~ /^([^"\*]*)[^:"]\/\//) {
-            checkwarn("CPPCOMMENTS",
-                      $line, length($1), $file, $l, "\/\/ comment");
-        }
+        #if($l =~ /^([^"\*]*)[^:"]\/\//) {
+        #    checkwarn("CPPCOMMENTS",
+        #              $line, length($1), $file, $l, "\/\/ comment");
+        #}
 
         my $nostr = nostrings($l);
         # check spaces after for/if/while/function call
